gemfiles/common.rb

```diff
@@ -2,17 +2,18 @@
 source 'https://rubygems.org'
 
 # dev dependencies:
-gem "pry",         "~> 0.13.1"
-gem "byebug",      "~> 11.1"
-gem "pry-byebug",  "= 3.9.0"
-gem "bump",        "~> 0.10"
-gem "minitest",    "~> 5.1"
-gem "minitest-rg", "~> 5.2"
-gem "mocha",       "~> 1.13"
-gem "rake",        "~> 13"
-gem "timecop",     "~> 0.9"
-gem "rubocop",     "~> 1.5.0"
-gem "rubocop-performance", "~> 1.10.2"
-gem "rubocop-rubycw",      "~> 0.1.6"
+gem "sqlite3"
+gem "pry"
+gem "byebug"
+gem "pry-byebug"
+gem "bump"
+gem "minitest"
+gem "minitest-rg"
+gem "mocha"
+gem "rake"
+gem "timecop"
+gem "rubocop"
```

Comment:
If we don’t commit lock files, and we don’t want every pull request to include RuboCop-induced style changes, then I recommend locking these gems to a minor version number.

Reply:
I bumped from 1.5 to 1.50 and only excluded ~5 things. This codebase just doesn't have the complexity or variance to matter imo.